### ISSUE TITLE
Client Server Mantid Pilot - Usage of back end server resources

### ISSUE THREAD

**Issue by NickDraper** [mantidproject/mantid#10125]
This issue was originally [TRAC 9282](http://trac.mantidproject.org/mantid/ticket/9282)

This ticket is blocked by : 
- [X] [TRAC9283](http://trac.mantidproject.org/mantid/ticket/9283)

Allow "remote worspace" and "remote algorithms" to store and process data on remote compute resources.

---

Keywords: SSC,2014,All, uiRemote

**Comment by martyngigg** [comment]
Not a current area of interest. Future work can be tracked in fresh issues.
